src/render/SDL_render.c

```diff
@@ -907,8 +907,7 @@ static SDL_RenderLineMethod SDL_GetRenderLineMethod()
 
 static void SDL_CalculateSimulatedVSyncInterval(SDL_Renderer *renderer, SDL_Window *window)
 {
-    /* FIXME: SDL refresh rate API should return numerator/denominator */
```

Comment:
I wonder what the pros and cons are for numerator/denominator versus a double. I suppose numerator/denom is more flexible and can keep more precise information when it's provided by the backend, whereas double might be more convenient for typical use cases?

Reply:
if int are 32bits, and double 64 bits then there are the information :)

Reply:
I don't think a double can perfectly represent the number `59.94` (a typical refresh rate), but it might not be important, I'm not sure.

Reply:
59.94 yes! but 1/3 Hz won't be ok... (or like 200/3 Hz to be more realistic if it exists ...)

Reply:
Just doing a bit of research: it looks like Unity has a RefreshRate struct which has numerator and denominator fields, and a convenience method to return a double-precision fractional value from dividing the two. Maybe that's a bit much for SDL though.

(A typical 59.94hz display is actually 60000/1001 hz, I believe.)

Reply:
yes, double won't represent fully 60000/1001

Reply:
(ironically my usb-c display just hanged)